service/service.go

```diff
@@ -87,6 +95,87 @@ func New(config Config) (*Service, error) {
 		}
 	}
 
+	var newOperatorBackOff *backoff.ExponentialBackOff
+	{
+		newOperatorBackOff = backoff.NewExponentialBackOff()
+		newOperatorBackOff.MaxElapsedTime = config.Viper.GetDuration(config.Flag.Service.Operator.OperatorBackOffDuration)
```

Comment:
Is this how it's supposed to be done? I am not sure what the proper place would be to get this info out of `config`

Reply:
See my other comment about validation of such information.

Reply:
Using fixed values here now.